### PR TITLE
Mijn 3236 tonk2

### PR DESCRIPTION
**Pull request by MokumJ**

### CODE REVIEW

File: src/server/services/focus/focus-types.ts [hunk 1]
```diff
@@ -112,7 +113,7 @@ export type DecisionFormatted =
   | 'buitenbehandeling';
 
 export type TextPartContents = (data: any, customData?: any) => string;
-export type LinkContents = (data: any, customData?: any) => Partial<LinkProps>;
```
Comment (timvanoostrom) [inline src/server/services/focus/focus-types.ts:115]: Why did you remove this?

**Review by timvanoostrom — CHANGES_REQUESTED** [review]

File: src/server/services/focus/focus-types.ts (hunk 1 above)
Comment (MokumJ) [reply src/server/services/focus/focus-types.ts:115]: I changed it into a none partial since LinkProps already has optional parameters. So I thought it would be ok to change it in just LinkProps

File: src/server/services/focus/focus-toxx-helpers.ts [hunk 2]
```diff
@@ -87,10 +87,7 @@ function getDocumentStepNotificationLink(
   document: FocusDocument,
   stepLabels: FocusStepContent
 ): LinkProps | undefined {
-  if (stepLabels.notification?.link) {
-    return stepLabels.notification?.link(document);
-  }
-  return undefined;
+  return stepLabels.notification?.link?.(document);
```
Comment (timvanoostrom) [inline src/server/services/focus/focus-toxx-helpers.ts:90]: This is not the correct syntax. It's sufficient to do: `return stepLabels.notification?.link(document);` If link isn't found on notification it won't do anything.

**Review by timvanoostrom — CHANGES_REQUESTED** [review]
Still a little thing to fix

File: src/server/services/focus/focus-toxx-helpers.ts (hunk 2 above)
Comment (MokumJ) [reply src/server/services/focus/focus-toxx-helpers.ts:90]: This gives a typescript error because cannot invoke a function when undefined. In typescript this is actually accepted syntax afaik.

Comment (timvanoostrom) [reply src/server/services/focus/focus-toxx-helpers.ts:90]: You are correct. I've misinterpreted the syntax.

**Review by timvanoostrom — APPROVED** [review]
LGTM
